tools/osbuilder/Makefile

```diff
@@ -9,7 +9,7 @@ ROOTFS_BUILDER := $(MK_DIR)/rootfs-builder/rootfs.sh
 INITRD_BUILDER := $(MK_DIR)/initrd-builder/initrd_builder.sh
 IMAGE_BUILDER  := $(MK_DIR)/image-builder/image_builder.sh
 
-DISTRO                := centos
+DISTRO                ?= centos
```

Comment:
@jing-wang4 do we really need to change this ?

Reply:
Found this issue when building initrd at osbuilder with DISTRO=fedora, it wont take DISTRO=fedora passed in, still centos instead.

```
$ sudo GOPATH=$GOPATH AGENT_INIT=yes LIBC=gnu DISTRO=fedora  make initrd
Creating rootfs for **centos**
/home/fedora/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/rootfs-builder/rootfs.sh -o 2.0.0-4de21e3d9598e8f035f229beeda1e94b7a7ee8f8 -r /home/fedora/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/centos_rootfs centos
INFO: Detecting go version
```

Reply:
This is interesting.  This bit also deserves its own patch, with that explanation in the patch.

Reply:
I see what's going on here, and I think you can just drop this change.
`DISTRO=fedora` shouldn't be an environment variable that the Makefile will try to make sense out of it.  Instead, it should be passed to make, like `make initrd DISTRO=fedora`.

Can you check that my suggestion would work for you *without* this patch?

Reply:
@jing-wang4, ^^^^

Reply:
Yes, working as expected per your suggestion. see updates: https://github.com/kata-containers/kata-containers/commit/b4ad61724999a4098f6a9f746701e46139e769c9